### DISCUSSION TITLE
Meetup Features

### DISCUSSION THREAD

**Discussion by grantnorwood** [remix-austin/remixaustin-com#59] [Ideas]
Let's proliferate some ideas around supporting our Meetup event!

@colbywhite already added some neat integration via the Meetup API to enable a dynamic button on our homepage, and there's other neat stuff we can do, too!

## Ideas

* **Check-in experience**
  * Ever get stuck outside the front door of HEB's Eastside Tech Hub waiting for someone to let you in?  What if we built some SMS integration to allow admins to enable "check-in" mode, where for a limited time attendees can visit the website and click "I'm here" button so that you don't stand at the door too long!?
* **Speaker sign-up**
  * List an array of interesting topics, and allow folks to volunteer to give a talk on one, or submit their own idea via a form.

_And other stuff ... let's discuss!_

**Comment by brookslybrand** [comment]
Calling bankruptcy on all discussions and issues today. Going to start revamping the direction of this project, so stay tuned 🚀
